app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -115,6 +115,7 @@ export const ContentForm: React.FC<IContentFormProps> = ({
     ...defaultFormValues(contentType),
     id: parseInt(id ?? '0'),
   });
+  const [product, setProduct] = React.useState('');
```

Comment:
Now we're maintaining an additional item in state that is essentially a duplicate of `productId`.

Reply:
Instead create a variable `isSummaryRequired` which will have the logic to determine whether the form field should be required.

Reply:
Updated, thanks